.gitignore

```diff
@@ -1,11 +1,10 @@
-src/app/colors/che-output-colors.constant.js
 src/app/colors/che-output-colors.constant.ts
-src/app/colors/codenvy-color.constant.js
 src/app/colors/codenvy-color.constant.ts
 src/app/colors/template/
-src/app/proxy/proxy-settings.constant.js
 src/app/proxy/proxy-settings.constant.ts
 src/app/proxy/template/
+# src/components/branding/branding.constant.ts
```

Comment:
Could you remove it?

Reply:
I've uncommented that line because `branding.constant.ts` should be ignored as generated on project build.

Reply:
It is a result file of the task 'branding'. We shouldn't commit it.